### ISSUE TITLE
default env $JAVA didn't work

### ISSUE THREAD

**Issue by xiaoping378** [zabbix/zabbix-community-docker#22]
Default environment JAVA is not work....

``` bash
[root@zabbix-server /]# /usr/local/sbin/zabbix_java/startup.sh
/usr/local/sbin/zabbix_java/startup.sh: line 62: /usr/lib/jvm/java-1.8.0-openjdk/bin/java: No such file or directory
Zabbix Java Gateway did not start

[root@zabbix-server /]# export JAVA=/usr/bin/java
[root@zabbix-server /]# 
[root@zabbix-server /]# 
[root@zabbix-server /]# /usr/local/sbin/zabbix_java/startup.sh
[root@zabbix-server /]# 
```

[root@zabbix-server /]#

**Comment by jangaraj** [comment]
Which Docker image version?

```
zabbix_server --version
```

**Comment by xiaoping378** [comment]
```
[root@zabbix-server /]# zabbix_server --version
Zabbix server v2.4.6 (2015-12-21) (revision {ZABBIX_REVISION}) (10 August 2015)
Compilation time: Dec 21 2015 23:49:55
```

**Comment by xiaoping378** [comment]
```
[root@zabbix-server /]# zabbix_server --version
Zabbix server v2.4.6 (2015-12-21) (revision {ZABBIX_REVISION}) (10 August 2015)
Compilation time: Dec 21 2015 23:49:55
```

**Comment by jangaraj** [comment]
Good, it's the latest image. I'll check.

**Comment by jangaraj** [comment]
/usr/bin/java will be used

```
[root@8982201fbbe5 /]# ls -lah /usr/bin/java
lrwxrwxrwx 1 root root 22 Jan 10 11:58 /usr/bin/java -> /etc/alternatives/java
[root@8982201fbbe5 /]# ls -lah /etc/alternatives/java
lrwxrwxrwx 1 root root 70 Jan 10 11:58 /etc/alternatives/java -> /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.65-3.b17.el7.x86_64/jre/bin/java
```

**Comment by xiaoping378** [comment]
yes,    the fix is working well.
But ..
Default the JAVA_HOME should point to the directory containing a bin/java.

**Comment by jangaraj** [comment]
Could you check docker image monitoringartist/zabbix-2.4 (only latest 2.4.7 is available there) and propose correct settings for java please? This image will be new community Zabbix image soon (PR #21).

**Comment by xiaoping378** [comment]
my suggestions:

``` shell
JAVA_HOME=/usr/lib/jvm/jre
JAVA=$JAVA_HOME/bin
```

~~JAVA=$JAVA_HOME/bin~~
sorry,  the env java is a wrong path.

**Comment by jangaraj** [comment]
Thank you. 

```
  JAVA_HOME=/usr/lib/jvm/jre \
  JAVA=/usr/lib/jvm/jre/bin \
```

Please retest image monitoringartist/zabbix-2.4

**Comment by xiaoping378** [comment]
Sorry,  java is not OK
~~JAVA=/usr/lib/jvm/jre/bin~~

Both of below  are OK.
- [x] JAVA=$JAVA_HOME/bin/java
- [x] JAVA=/usr/bin/java

**Comment by jangaraj** [comment]
No problem.

```
JAVA_HOME=/usr/lib/jvm/jre \
JAVA=/usr/bin/java \
```

Please recheck monitoringartist/zabbix-2.4.

**Comment by xiaoping378** [comment]
OK, work well

**Comment by jangaraj** [comment]
Fixed in new Docker image zabbix/zabbix-2.4 (zabbix/zabbix-3.0).
